doc/getting-started/getting-started.rst

```diff
@@ -300,9 +292,9 @@ Generate a Board Configuration File
 
       .. code-block:: bash
 
-         make clean && make board_inspector
+         debian/debian_build.sh clean && debian/debian_build.sh board_inspector
```

Comment:
I see these error messages about missing files when I try building the board_inspector with this command.  Is this correct and expected?

```
david@david-X220:~/acrn-work/acrn-hypervisor$ debian/debian_build.sh board_inspector
CONFIGDIRS        = 'misc/config_tools/data'
gbp:info: Changelog last touched at 'ed9cc9409a49c762273775314ae4fdcfff56543d'
gbp:info: Continuing from commit 'ed9cc9409a49c762273775314ae4fdcfff56543d'
gbp:info: Changelog unknown~1.gbp808b4f (snapshot #1) prepared up to 808b4f6
debuild -eCONFIGDIRS='misc/config_tools/data' -- binary-indep
 dpkg-buildpackage --rules-target binary-indep -us -uc -ui
dpkg-buildpackage: info: source package acrn-hypervisor
dpkg-buildpackage: info: source version 3.2-unstable~1.gbp808b4f
dpkg-buildpackage: info: source distribution UNRELEASED
dpkg-buildpackage: info: source changed by David Kinder <david.b.kinder@intel.com>
 fakeroot debian/rules binary-indep
   dh_update_autotools_config -i
   dh_autoreconf -i
   dh_auto_configure -i
   debian/rules override_dh_auto_build-indep
   build/documentation/html/index.html: BUILD documentation
   override_dh_auto_build-indep: BUILD acrn-board-inspector
I: pybuild base:239: /usr/bin/python3 setup.py build 
package init file '../../misc/config_tools/board_inspector/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/acpiparser/aml/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/extractors/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/inspectorlib/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/legacy/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/schema/__init__.py' not found (or not a regular file)
   dh_auto_test -i
   create-stamp debian/debhelper-build-stamp
   dh_testroot -i
   dh_prep -i
   debian/rules override_dh_auto_install-indep
   override_dh_auto_install-indep: INSTALL acrn-board-inspector
I: pybuild base:239: /usr/bin/python3 setup.py install --root /home/david/acrn-work/acrn-hypervisor/debian/python3-acrn-board-inspector 
/usr/lib/python3/dist-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
package init file '../../misc/config_tools/board_inspector/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/acpiparser/aml/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/extractors/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/inspectorlib/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/legacy/__init__.py' not found (or not a regular file)
package init file '../../misc/config_tools/board_inspector/schema/__init__.py' not found (or not a regular file)
   override_dh_auto_install-indep: INSTALL documentation
   dh_install -i
   dh_installdocs -i
   dh_installchangelogs -i
   dh_python3 -i
   debian/rules override_dh_installsystemd
   dh_lintian -i
   dh_perl -i
   dh_link -i
   dh_strip_nondeterminism -i
   dh_compress -i
   dh_fixperms -i
   debian/rules override_dh_missing
   debian/rules override_dh_installdeb
   debian/acrn-hypervisor.postinst: CREATE debian/acrn-hypervisor.postinst
   debian/acrn-hypervisor.postrm: CREATE debian/acrn-hypervisor.postrm
   debian/acrn-hypervisor.prerm: CREATE debian/acrn-hypervisor.prerm
   dh_gencontrol -i
   dh_md5sums -i
   dh_builddeb -i
dpkg-deb: building package 'acrn-doc' in '../acrn-doc_3.2-unstable~1.gbp808b4f_all.deb'.
dpkg-deb: building package 'python3-acrn-board-inspector' in '../python3-acrn-board-inspector_3.2-unstable~1.gbp808b4f_all.deb'.
/home/david/acrn-work/acrn-hypervisor
```

Reply:
Yes, these warning messages are known build warnings and could be ignored：https://github.com/projectacrn/acrn-hypervisor/blob/master/debian/README.rst#known-build-warnings

Reply:
Thanks for pointing that out. We currently don't publish any of these debian packaging documents as part of the project ACRN documentation (docs.projectacrn.org).  Looks like we'll need to do that.  @NanlinXie

Reply:
Yes, we could publish the debian packageing document based on current README.rst, let's discuss more details later.